src/pages/settings/wallet-group-settings/wallet-group-settings.html

```diff
@@ -1,26 +1,45 @@
-<ion-header>
-  <ion-navbar>
-    <ion-title *ngIf="canSign">{{'Wallet Key Settings' | translate}}</ion-title>
-    <ion-title *ngIf="!canSign">{{'Read Only Wallets' | translate}}</ion-title>
+<ion-header class="wide-header">
+  <ion-navbar [navbar-bg]="headerColor">
+    <ion-title>
+      <div [reveal-at-scroll-pos]="expandableHeader.headerHeight" [scrollArea]="scrollArea">
+        <span *ngIf="canSign">{{'Key Settings' | translate}}</span>
+        <span *ngIf="!canSign">{{'Read Only Key' | translate}}</span>
+      </div>
+    </ion-title>
+    <ng-content select="[right]"></ng-content>
   </ion-navbar>
 </ion-header>
-<ion-content>
+
+<ion-content #scrollArea [fixed-scroll-bg-color]="headerColor" [ion-content-background-color]="headerColor">
+  <div class="wrapper">
+    <expandable-header [scrollArea]="scrollArea" [fadeFactor]="5" [disableFade]="true" #expandableHeader>
+      <ion-toolbar [navbar-bg]="headerColor" class="wide-header__title">
+        <expandable-header-primary>
+          <span *ngIf="canSign">{{'Key Settings' | translate}}</span>
+          <span *ngIf="!canSign">{{'Read Only Key' | translate}}</span>
+          <div class="subtitle" *ngIf="wallets">
+            This key contains {{wallets.length}} wallets.
```

Comment:
We should not show this message to read only. Since they can come from multiple different keys

Reply:
Done. Thanks!